gpii/node_modules/deviceReporter/test/StaticDeviceReporter.js

```diff
@@ -0,0 +1,111 @@
+/*!
```

Comment:
We should put "Tests" into the name of this file so it doesn't cause confusion in people's IDEs ....